pages/common/git-info.md

```diff
@@ -0,0 +1,13 @@
+# git info
+
+> Display Git repository information.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-info>.
+
+- Display repository information about remote locations, remote and local branches, most recent commit data and `.git/config` settings:
```

Comment:
```suggestion
- Display remote locations, remote and local branches, most recent commit data and `.git/config` settings:
```
I think this is equally informative. The command doesn't really print information about those things, but lists the things themselves.